examples/fabric-admin/rpc/RpcServer.cpp

```diff
@@ -147,11 +174,12 @@ class FabricAdmin final : public rpc::FabricAdmin, public IcdManager::Delegate
         if (error == CHIP_NO_ERROR)
         {
             NodeId nodeId = DeviceManager::Instance().GetNextAvailableNodeId();
```

Comment:
nit, why not just assign to mNodeId directly. Is it possible that before `DeviceManager::Instance().PairRemoteDevice(nodeId, code.c_str());` is called that mNodeId changes? If not I would remove nodeId